content/kotlin/concepts/functions/extension-function.md

```diff
@@ -0,0 +1,80 @@
+## Extension Functions 
```

Comment:
Originally, I incorrectly reviewed this as an edit to existing entry. But, this is being added as a completely new doc and the path should be -
content/kotlin/concepts/extension-functions/extension-functions.md

Please update the folder and filename accordingly

Reply:
This is also missing the metadata section please use another entry as a reference or copy and edit the metadata from one of our templates
https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md?plain=1